cartographer/io/internal/mapping_state_serialization.cc

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2018 The Cartographer Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "cartographer/io/internal/mapping_state_serialization.h"
+#include "cartographer/mapping/proto/serialization.pb.h"
+#include "cartographer/transform/transform.h"
+
+namespace cartographer {
+namespace io {
+
```

Comment:
super-nit: remove the empty line between `namespace io {` and `namespace {`.

Reply:
done.